config/routes.rb

```diff
@@ -108,29 +105,29 @@
       post '/signup', action: :create, as: :user_registration
     end
 
-    get '/dashboard' => 'dashboard#index', as: :dashboard
-    get '/dashboard/:filter' => 'dashboard#index', as: :dashboard_filter
+    get '/dashboard', to: 'dashboard#index', as: :dashboard
+    get '/dashboard/:filter', to: 'dashboard#index', as: :dashboard_filter
 
     # settings
-    get    '/settings' => 'users#edit', as: :edit_user
-    patch  '/settings' => 'users/registrations#update'
-    get    '/settings/email' => 'users/registrations#edit_email', :as => :edit_user_email
-    get    '/settings/profile' => 'users/profiles#edit', :as => :edit_user_profile
-    get    '/settings/notifications' => "users/notifications#edit", :as => :settings_notifications
-    patch  '/settings/notifications' => "users/notifications#update"
+    get    '/settings',               to: 'users#edit', as: :edit_user
+    patch  '/settings',               to: 'users/registrations#update'
+    get    '/settings/email',         to: 'users/registrations#edit_email', as: :edit_user_email
+    get    '/settings/profile',       to: 'users/profiles#edit', as: :edit_user_profile
+    get    '/settings/notifications', to: "users/notifications#edit", as: :settings_notifications
+    patch  '/settings/notifications', to: "users/notifications#update"
 
     namespace :users, path: 'user' do
       resource :balance, only: [:show] { post :withdraw }
       resource :payment_option, only: [:show, :create, :update]
       resource :tax_info, only: [:show, :create, :update] do
-        get ':form_type' => 'tax_infos#show'
+        get ':form_type', to: 'tax_infos#show'
       end
     end
 
     # Confirmation
-    get    '/users/confirmation/new' => 'users/confirmations#new', :as => :new_user_confirmation
-    get    '/users/confirmation' => 'users/confirmations#show', :as => :user_confirmation
-    post   '/users/confirmation' => 'users/confirmations#create'
+    get    '/users/confirmation/new', to: 'users/confirmations#new', as: :new_user_confirmation
```

Comment:
Line is too long. [95/80]<br>Unnecessary spacing detected.<br>Put one space between the method name and the first argument.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.